Berksfile

```diff
@@ -2,4 +2,10 @@ source 'https://supermarket.chef.io'
 
 metadata
 
+case RUBY_VERSION
+when '2.1.6'
```

Comment:
As you [mentioned](https://github.com/cerner/cerner_splunk/pull/140#pullrequestreview-23046625), this chef version (12.4.3) is locked in the Gemfile for this specific ruby version (2.1.6).